index.html

```diff
@@ -44,9 +44,7 @@
         </p>
       </div>
 
-      <div class="connect">
-        <p>LET’S CONNECT</p>
-      </div>
+      <p class="connect">LET’S CONNECT</p>
 
       <div class="social-icons">
         <span class="iconify" data-icon="iconoir:github"></span>
```

Comment:
- [ ] For displaying social icons... I think it would be better if you use the `ul` tag instead of `div`. 

```
As the requirement says:  
The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)
```